core/vm/analysis.go

```diff
@@ -26,6 +26,20 @@ func codeBitmap(code []byte) []uint64 {
 	for pc := 0; pc < len(code); {
 		op := OpCode(code[pc])
 		pc++
+		// Short circruit for now on EOF ops with immediates.
+		// TODO(matt): make EOF-specific code bitmap
+		if op == RJUMP || op == RJUMPI || op == CALLF {
+			// TODO(CZ): For now the logic of bitvec.setN has been copied
```

Comment:
@ziogaschr Is this still a TODO? 

Ref https://github.com/ethereum/go-ethereum/pull/26133/files#diff-882af11429522494f308c57017fe1c44e86b338e3bf5c308584d9d187606d252R75